pyproject.toml

```
@@ -40,15 +40,16 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.9,<3.13"
-numpy = "^1.24"
+python = "^3.9,<3.14"
+numpy = "^2"
 scipy = "^1.10.1"
 PyYAML = "^6.0"
 lz4 = "^4.0.2"
-numba = "^0.59.0"
+# 0.60.0 is the last version compatible with 3.9 and 0.61 the first with 3.13
+numba = "~0.60.0 || ~0.61.0"
 packaging = "^24.1"
 # ekomark
-banana-hep = { version = "^0.6.12", optional = true }
+banana-hep = { git = "https://github.com/NNPDF/banana", branch = "relax_restrictions", optional = true}
```

```suggestion
banana-hep = { version = "^0.6.16", optional = true}
```
+ `poetry lock` of course